id_dsl.rb

```diff
@@ -19,6 +19,7 @@ def create_driver(headless)
   driver_options = Selenium::WebDriver::Chrome::Options.new
   driver_options.add_argument('--disable-popup-blocking')
   driver_options.add_argument('--disable-translate')
+  driver_options.add_argument('--ignore-certificate-errors')
```

Comment:
I had this in earlier, but the CISO from VWDS got in touch to ask to remove the line from the code for security reasons.

Reply:
Ops, no worries, going to close the pull request :)